typings/internal-ambient.d.ts

```diff
@@ -258,6 +258,7 @@ declare namespace NodeJS {
     _firstFileName?: string;
 
     helperExecPath: string;
+    readonly resourcesPath: string;
```

Comment:
this is already in electron.d.ts via https://github.com/electron/electron/blob/main/docs/api/process.md#processresourcespath-readonly

Reply:
@miniak Thank you very much! I didn't notice that the `electron.d.ts` file got generated with the needed types